src/box/memtx_engine.c

```diff
@@ -78,6 +78,57 @@ enum {
 	MAX_TUPLE_SIZE = 1 * 1024 * 1024,
 };
 
+static inline void
+memtx_free_tuple_from_lifo(struct memtx_engine *memtx, void *item)
```

Comment:
Function called _from_lifo but in fact it has nothing to do with lifo. I suggest you to rename it to memtx_tuple_free or sort of and use it in memtx_tuple_delete as well.

Reply:
Fixed